trimChamber.py

```diff
@@ -26,6 +26,8 @@ def runCommand(cmd):
 
 from qcoptions import parser
 
+parser.add_option("--trimRange", type="string", dest="rangeFile", default="",
```

Comment:
Same as above, don't set a default, then later do a check on being set, and a ```try```/```except``` to ensure the file exists.